src/quick-lint-js/parse.h

```diff
@@ -408,13 +408,30 @@ class parser {
   template <QLJS_PARSE_VISITOR Visitor>
   void visit_compound_assignment_expression(expression_ptr lhs,
                                             expression_ptr rhs, Visitor &v) {
-    this->visit_expression(lhs, v, variable_context::rhs);
+    // bool used = false;
+    // used = this->visit_expression(lhs, v, variable_context::rhs);
+    // this->visit_expression(rhs, v, variable_context::rhs);
+    // this->maybe_visit_assignment(lhs, v, used);
+    bool used = false;
```

Comment:
Minor: I don't understand what `used` means. Perhaps `lhs_is_variable` is a better name? Or maybe we don't need the variable at all, and we can write `if (lhs->kind() == expression_kind::variable) {` on line 424?